modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -239,4 +298,22 @@ private static <T extends Comparable<T>> void updateTrackerIgnoringTrackerClosed
             // No-op.
         }
     }
+
+    private void cleanupSnapshots() {
```

Comment:
This method (and a couple of other methods) are copy-pasted from `SnapshotAwarePartitionDataStorage`. Would it make sense to break `SnapshotAwarePartitionDataStorage` in two parts and reuse one part here?